### ISSUE TITLE
[completely offtopic] inputs on helm/stable/graylog

### ISSUE THREAD

**Issue by apenjag** [tao-zhang/jenkins-dsl-jobs#3]
Hello,

apologies for the offtopic comment but i didn't know how to reach you. I've seen your issue on stable/graylog for defining inputs and as i'm right now in the same situation, maybe you have found a way to do it.

Hope you can point me in the right direction.
Thanks!

Best
Alessandro

**Comment by tao-zhang** [comment]
@apenjag  Do you mean the issue I created here https://github.com/helm/charts/issues/18153? If so, you can create an ingress TCP service, for example https://github.com/tao-zhang/eks/blob/330b70ade3d4318e6e5e195d19b455ea2c4a8758/helm/nginx-ingress/values.yaml#L574

**Comment by apenjag** [comment]
Oh ok. So that is to create the nginx-ingress and then you keep the graylog service into the cluster (ClusterIP) and also the inputs as ClusterIP services? Or Nodeport ?
I created with the graylog chart a ELB loadbalancer in AWS but there was no way to use that ELB for inputs too (or at least I wasn't able to find a way).

Thanks a lot for you prompt answer! very appreciated!

**Comment by tao-zhang** [comment]
No, this is a Ingress TCP service, the backend is still a service from graylog which you should create it in graylog. For example:

```
input:
    tcp:
      service:
        type: ClusterIP
      ports:
      - name: syslog
        port: 6514
```
